src/decision/dMCDA.jl

```diff
@@ -182,6 +187,15 @@ function mcda_normalize(x::Matrix)::Matrix
     return x ./ sqrt.(sum(x .^ 2, dims=1))
 end
 
+"""
+    mcda_normalize(x::DataFrame)::DataFrame
+
+Normalize weights for a set of scenarios (wse/wsh) for MCDA.
+"""
+function mcda_normalize(x::DataFrame)::DataFrame
+    norm_x = sum.(eachrow(x))
+    return x ./ norm_x
+end
```

Comment:
Did you compare this with the previous approach converting to matrix?

Reply:
The Matrix version took 0.406 seconds, the eachrow() version took 0.082 seconds.

As I said in my email, I think this difference is due to the converison to Matrix taking time. I'm guessing eachrow is also optimised for use with dataframes.

Reply:
How did you do the timing and did you check allocations?

Reply:
I assigned both versions to a function and did @time func(x)... I know its preferable to use Benchmarking Tools but I just did it quickly

Reply:
If you're okay with that then let's roll with it

Reply:
I'm okay with it, but are you?